packages/node_modules/@ciscospark/react-container-activity-list/src/formatters/at-mention.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {base64} from '@ciscospark/common';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './at-mention.css';
+
+const EVENT_NAME_MENTION_CLICKED = `mention:clicked`;
+
+function AtMentionComponent({
+  content,
+  onEvent
+}) {
+
+  function isMention(target) {
+    return target.tagName === `SPARK-MENTION` && target.dataset && typeof onEvent === `function`;
+  }
+
+  function handleClick(e) {
+    triggerEvent(e);
+  }
+
+  function handleKeyUp(e) {
+    if (e.keyCode && e.keyCode === 13) {
+      return triggerEvent(e);
+    }
+    return false;
+  }
+
+  function triggerEvent(e) {
+    const target = e.target;
+    if (isMention(target)) {
+      onEvent(EVENT_NAME_MENTION_CLICKED, {
```

Comment:
Should we check if onEvent exists first?

Reply:
It's a bit obtuse but the `onEvent` check is in the `isMention()` check.